ecr-login/config/seelog.go

```diff
@@ -15,46 +15,32 @@ package config
 
```

Comment:
Can we rename this file to reflect the updated code?

Reply:
Yes

Reply:
Done